pagopa/io-app
# fix: [IABT-1429] QR code library updated

## Short description
Updated rn-qr-generator library.
Updated RNQrGeneratorModule.java to allow user to read QRcode images of any size from gallery without crash or errors on Android.

[PR rn-qr-generator
](https://github.com/gevgasparyan/rn-qr-generator/pull/28)
## How to test
Upload a large QRcode image from the gallery and check that everything works, both with a valid QRcode and with an invalid QRcode.
